README.md

```diff
@@ -169,6 +171,12 @@ function getAllAddons({ categoryId }: GetAllAddonsParams = {}) {
   the `yarn flow type-at-pos ...` command to trace which types are being applied
   to the code. See `yarn flow -- --help type-at-pos` for details.
 
+### Prettier
+
+We Use [Prettier][] to automatically format our JavaScript code and stop all the
```

Comment:
Did you mean to say "We use [Prettier][] ..."?

Reply:
Ah, yes, that was a typo